Sources/VRMSceneKit/CustomType/VRMSpringBone.swift

```diff
@@ -12,32 +12,32 @@ import VRMKit
 
 final class VRMSpringBone {
     struct SphereCollider {
-        let position: SCNVector3
-        let radius: SCNFloat
+        let position: simd_float3
```

Comment:
Swift 5からSwiftらしい書き方が可能になったため、新しい実装に関してはそちらを利用していただけると助かりますー 🙏 
```suggestion
        let position: SIMD3<Float>
```